src/html/layout.html

```diff
@@ -55,7 +55,11 @@ <h1 class="logo-holder">
     </div>
   </main>
 
-  <footer class="footer"></footer>
+  <footer class="footer">
+    <div class="container">
+      {% include './components/social-share.html' %}
```

Comment:
tak ogólnie to nie wiem czy to powinien być footer - mi się wydaje, że to jedno z CTA i bym wrzucił w navbar.